modules/configuration/src/main/java/org/apache/ignite/internal/configuration/util/ConfigurationUtil.java

```diff
@@ -630,7 +631,6 @@ private static Map<Class<?>, Set<Class<?>>> schemaExtensions(
      * PolymorphicId}.
      *
      * @param schemaClass Configuration schema class.
-     * @return Schema fields.
```

Comment:
why did you remove this tag?

Reply:
They duplicate information from the description.
Do you want me to return them?

Reply:
yes, let's keep them

Reply:
Yes, I want to delete.

Reply:
ok =)